### PR TITLE
#174566480 Configure babel to transpile code

### PR DESCRIPTION
**Pull request by Bernard-Namangala**
## What does this PR do?
This PR configures babel to transpile the es6 code 

## Description of tasks to be completed?
- install @babel/node @babel/core @babel/preset-env @babel/cli
- create .babelrc file and add configurations
- add 3000 as default port for express server

## How should this be manually tested/checked?
- Clone the PR
- Cd into the branch `ch-configure-babel-174566480`
- Install the dependencies using `npm install`
- Run the server on the terminal using `npm run start`
- It can be viewed on this link: http://127.0.0.1:3000/

## What are the relevant pivotal tracker stories?
[#174566480](https://www.pivotaltracker.com/n/projects/2463429/stories/174566480)

### CODE REVIEW

**Review by Extremophile-ai — DISMISSED** [review]

**Review by Fiyin-Anne — COMMENTED** [review]
Works fine.

**Review by Fiyin-Anne — APPROVED** [review]

**Review by Fiyin-Anne — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -4,13 +4,18 @@
   "description": "",
   "main": "app.js",
   "dependencies": {
+    "@babel/core": "^7.11.5",
+    "@babel/node": "^7.10.5",
+    "@babel/preset-env": "^7.11.5",
```
Comment (Endowmissy) [inline package.json:9]: @babel/core @babel/node @babel/preset-env should be under devDependencies, you need to fix this

File: package.json [hunk 2]
```diff
@@ -4,13 +4,18 @@
   "description": "",
   "main": "app.js",
   "dependencies": {
+    "@babel/core": "^7.11.5",
```
Comment (Endowmissy) [inline package.json:7]: @babel/core should be under devDependencies, you need to fix this

File: package.json [hunk 3]
```diff
@@ -4,13 +4,18 @@
...
+    "@babel/node": "^7.10.5",
```
Comment (Endowmissy) [inline package.json:8]: @babel/node should also be under devDependencies

**Review by codeliezel — COMMENTED** [review]
@Bernard-Namangala Good job. re-install all the babel dependencies to under devDependencies.

**Review by Fiyin-Anne — APPROVED** [review]

**Review by Fiyin-Anne — COMMENTED** [review]
Are the packages supposed to remain under `dependencies`?

**Comment by codeliezel** [comment]
> Are the packages supposed to remain under `dependencies`?

@Fiyin-Anne go and read more about how some packages are meant to be installed in the general dep or devdep.

**Comment by Fiyin-Anne** [comment]
> > Are the packages supposed to remain under `dependencies`?
> 
> @Fiyin-Anne go and read more about how some packages are meant to be installed in the general dep or devdep.

Alright. Will do.

**Comment by codeliezel** [comment]
@Bernard-Namangala before I approve, write your PT story following the convention @Fiyin-Anne  and comment here to let me know.

**Review by codeliezel — COMMENTED** [review]
Good job all the way

**Review by bellogo — APPROVED** [review]
Code meets requirement after testing.

**Comment by Bernard-Namangala** [comment]
> @Bernard-Namangala before I approve, write your PT story following the convention @Fiyin-Anne and comment here to let me know.
i have added the description to pt story and i have also attached a label

**Comment by codeliezel** [comment]
> > @Bernard-Namangala before I approve, write your PT story following the convention @Fiyin-Anne and comment here to let me know.
> > i have added the description to pt story and i have also attached a label

Good

**Pull request merged**